home/ivp-planning/advanced-tools/local-library.md

```diff
@@ -0,0 +1,55 @@
+# Local Library
+
+  You can import your own files/objects from your PC into iVP by using the "Import" option, found under the "File" menu at the top of the application.  
+
+![](../../../.gitbook/assets/3d-import-file.png)
+ 
+
+When importing your file, you have the option to change its parameters, as well as the option to add it to your library, which then will show up at the bottom of the default library on the left side of the application.
+
+![](../../../.gitbook/assets/3d-import-addtolibrary.png)
+
+
+### iVP Library
+Once you have added an item to your library, the folder "iVP Library" will appear at the bottom of the library window.
+From now on every item that you import, and chose to add to your library, can be found inside the "iVP Library" folder.  
+
+The Order of your items can only be changed inside the directory itself, **not** inside iVP Planning.
+
+![](../../../.gitbook/assets/3d-import-library.png)
+
+### Editing your iVP Library
+
+When hovering an entry in the library, a button with a folder icon will appear. Clicking it will open the respective folder or select the respective file in your file browser  
+![](../../../.gitbook/assets/3d-import-ivplibrary.png)  
+The folders name can be changed at any time.  
+Supported file-types include Images, .plan and 3D-Models.  
+
+
+Your "iVP Library" folder will be created in the default iVP-Directory.  
+You can click on the folder button (highlighted below) next to any object inside the folder to open the directory.
+
+![](../../../.gitbook/assets/3d-import-openobject.png)
+
+You can create your own iVP library folder in whichever directory you want to, by selecting the "Libraby" tab in the iVP settings, found at the top of the application.
+
+![](../../../.gitbook/assets/3d-import-library-changepath.png)  
+
+### Moving Files to and from your personal Library
+
+Instead of importing files via the "Import" option, you also have the option to move your files by placing them into your personal iVP Library.  
+Once your files are inside your library folder, they will show up in iVP Planning, and can be placed from your personal library.
+
+You can freely organize your files inside your library. You can create new folders and move files between them. The library in Planning will resemble the folder structure you create on your computer.  
+
+The files that need to be moved are:  
```

Comment:
Needs a bit more context. Explain that additional files will be created and that you have to move them together. Keep the rest as is.